fvm/src/call_manager/default.rs

```diff
@@ -412,6 +405,62 @@ where
         Ok(())
     }
 
+    fn upgrade_actor<K: Kernel<CallManager = Self>>(
+        &mut self,
+        actor_id: ActorID,
```

Comment:
We can pass in the "caller" here to preserve it.  It's kind of strange, but I don't think we have any better options.